operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,316 @@
+# Manage Critical Services
+
+* [Overview](#overview)
+* [Procedure to add a critical service from the ConfigMap](#procedure-to-add-a-critical-service-from-the-configmap)
+* [Procedure to delete a critical service from the ConfigMap](#procedure-to-delete-a-critical-service-from-the-configmap)
+* [Procedure to modify a critical service from the ConfigMap](#procedure-to-modify-a-critical-service-from-the-configmap)
+
+## Overview
+
+This page contains the procedures to list, add, delete and modify the critical services.
+
+The ConfigMap `rrs-mon-static` in the `rack-resiliency` namespace is where RR keeps its list of critical services. The API/CLI commands to add services end up adding 
+the new services to this ConfigMap. But because the API/CLI does not support edits or deletes, those can only be accomplished by directly editing the ConfigMap. 
+
+Please note that having two services with the same name in different namespaces is generally not considered a best practice in CSM, and as such, this use case is not supported in RRS. Additionally, services with the same name but different types (e.g., `statefulset` and `deployment`) are also not supported. Please ensure that this guideline is followed when adding a new critical service.
+
+## Procedure to add a critical service from the ConfigMap
+
+Refer "Updating critical services" section in [Adding critical service(s)](https://github.com/Cray-HPE/docs-csm/blob/CASM-5612/operations/rack_resiliency/rack_resiliency_cli.md#updating-critical-services)
+
+## Procedure to delete a critical service from the ConfigMap
+
+### 1. Verify that the ConfigMap exists
+
+```bash
+kubectl get ConfigMaps -n rack-resiliency
+```
+
+**Example output:**
+
+```text
+NAME                 DATA   AGE
+istio-ca-root-cert   1      6d21h
+kube-root-ca.crt     1      6d21h
+rrs-mon-dynamic      2      3d4h
+rrs-mon-static       11     3d4h
+```
+
+### 2. Verify that the list of critical services is present in the ConfigMap
```

Comment:
Like with the earlier step, if this step is to verify that the list is present, then we need to say what they should do if it is not. Instead, if this step is just to view its current contents, it should say that. Or probably it would be better to make that a parallel procedure to the Delete and Edit procedures -- a procedure to view the current critical services list in the configmap. For the delete procedure, they just need to edit the configmap -- they don't need to list it first.